mne/preprocessing/ica.py

```diff
@@ -1018,7 +1023,10 @@ def find_bads_eog(self, inst, ch_name=None, threshold=3.0,
         if len(scores) == 1:
             scores = scores[0]
 
-        return eog_idx, scores
+        if not hasattr(self, 'labels_'):
+            self.labels_ = {}
+        self.labels_['eog'] = list(eog_idx)
+        return self.labels_['eog'], scores
```

Comment:
So this is the method for finding eog artifacts, so we should have an expplicit label for that. if it corresponds to the family of bads as defined by another method that's fine, no?